helm/ingress-azure/templates/deployment.yaml

```diff
@@ -50,6 +50,10 @@ spec:
             port: {{ .Values.kubernetes.httpServicePort }}
           initialDelaySeconds: 15
           periodSeconds: 20
+        {{- with .Values.resources }}
```

Comment:
can you add a helm render test as well ?
You can reference [this commit](https://github.com/Azure/application-gateway-kubernetes-ingress/commit/3fedb57daec00c0ac78f61714923567dfbdb0b75).
When run the test, first set `export RENDER_SNAPSHOTS="true"`.
Then execute `go test -tag=unittest ./...` in the `./helm/ingress-azure/tests` folder to generate the YAML.
Then you can check-in the updated YAML.